examples/pytorch/grand/utils.py

```diff
@@ -0,0 +1,17 @@
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+
+def consis_loss(logps, temp, lam):
```

Comment:
Is this function from the author? If so, pls add credit of the original authors.